### PR TITLE
[fix] s3 user agent configuration is incorrect

### PR DESCRIPTION
**Pull request by Ziy1-Tan**
### What problem does this PR solve?

Issue Number: https://github.com/opencurve/curve/issues/2898

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Ziy1-Tan** [comment]
cicheck

**Comment by Ziy1-Tan** [comment]
cicheck

File: test/common/string_util_test.cpp [hunk 1]
```diff
@@ -59,5 +59,46 @@ TEST(Common, StringToUll) {
...
+}
+
+TEST(StringUtilTest, trim) {
+  std::array<std::array<std::string, 2>, 6> cases = {
+      std::array<std::string, 2>{"hello", "hello"},
+      std::array<std::string, 2>{"   hello", "hello"},
+      std::array<std::string, 2>{"hello    ", "hello"},
+      std::array<std::string, 2>{"    hello  ", "hello"},
+      std::array<std::string, 2>{"    ", ""},
+      std::array<std::string, 2>{"", ""},
```
Comment (aspirer) [inline test/common/string_util_test.cpp:94]: we should add a case for middle space, such as "   S3 Browser  ", "S3 Browser", the expect trimed string is "S3 Browser".

**Review by aspirer — CHANGES_REQUESTED** [review]
good job, but one more testcase can be added.

File: test/common/string_util_test.cpp (hunk 1 above)
Comment (Ziy1-Tan) [reply test/common/string_util_test.cpp:102]: Fixed.

**Review by aspirer — COMMENTED** [review]
good

**Comment by caoxianfei1** [comment]
@Ziy1-Tan squash into one commit.

**Comment by aspirer** [comment]
> @Ziy1-Tan squash into one commit.

This is not a requirement, it is a suggestion.

**Comment by aspirer** [comment]
cicheck

**Comment by Ziy1-Tan** [comment]
cicheck

**Comment by Ziy1-Tan** [comment]
cicheck

**Review by aspirer — APPROVED** [review]
LGTM~

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**